LongWarOfTheChosen/Src/LW_Overhaul/Classes/UIStrategyMapItem_Mission_LW.uc

```diff
@@ -337,6 +362,69 @@ simulated function OnLoseFocus()
 	super.OnLoseFocus();
 }
 
+// KDM : Tooltips need to be regenerated, on item focus, so that we get up to date information regarding mission expiration and infiltration percentage.
+//
+// It is IMPORTANT to note that the mission expiration and infiltration percentage will not update while the mission is selected; in other words, it will
+// not update when the user is inactive, but the geoscape time is slowly moving forward.
+//
+// The alternative was to regenerate the tooltip within UpdateFromGeoscapeEntity(), as this function is continuously called; however, I didn't want to
+// bombard the tooltip manager with too many update calls. Additionally, time creeps forward so slowly, when the user is inactive, that the mission 
```

Comment:
It creeps forward slowly, but if a player wanders away and then comes back a few minutes later, there will be a discrepancy in the infiltration percentages shown, correct? That may be a cost worth paying to avoid an even laggier Geoscape in the end game, but worth bearing in mind.

Reply:
Yes - it can create an inconsistency in display. I originally had the code in UpdateFromGeoscapeEntity(); however, I wasn't sure how expensive repeatedly recreating a tooltip was. Want to give it a try? If so, we can always change it back if any issues arise.

Reply:
Yes, let's try having it in `UpdateFromGeoscapeEntity()` and hope we remember the potential cause if things seem laggy down the road 😆

Reply:
I've run into a bit of a snag - didn't work like I'd thought. Looking into it.

Reply:
Ok - I thought I had tooltips updating via UpdateFromGeoscapeEntity() beforehand, but I thought wrong. Finally figured out how to update it on the fly. Generally speaking, looks like tooltips are meant to be updated once, when an entity gains focus/selection.

Nonetheless, seems to work well now.